app/components/error_list/error_list.js

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PropTypes, PureComponent} from 'react';
+import {
+    Dimensions,
+    StyleSheet,
+    View,
+    TouchableOpacity
+} from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+import FormattedText from 'app/components/formatted_text';
+
+import GeneralError from './general_error';
+
+const {width: deviceWidth} = Dimensions.get('window');
+
+const style = StyleSheet.create({
+    closeButton: {
+        height: 20,
+        width: 20,
+        borderRadius: 10,
+        marginBottom: 5,
+        borderColor: '#fff',
+        borderWidth: 1,
+        alignItems: 'center',
+        justifyContent: 'center'
+    },
+    closeButtonContainer: {
+        alignItems: 'center',
+        justifyContent: 'center',
+        marginTop: 5
+    },
+    closeButtonText: {
+        color: '#fff'
+    },
+    container: {
+        paddingTop: 15,
+        paddingBottom: 15,
+        alignItems: 'center',
+        justifyContent: 'center',
+        minHeight: 75
+    },
+    wrapper: {
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        width: deviceWidth,
+        overflow: 'hidden',
+        backgroundColor: 'rgba(255, 116, 92, 1)',
+        zIndex: 99999
+    }
+});
+
+export default class ErrorList extends PureComponent {
+    static propTypes = {
+        actions: PropTypes.shape({
+            dismissError: PropTypes.func.isRequired
+        }).isRequired,
+        errors: PropTypes.array.isRequired
+    }
+
+    render() {
+        return (
+            <View style={[style.wrapper, (!this.props.errors.length && {height: 0})]}>
```

Comment:
This would be a great configuration. On mobile, debugging is hard enough in the wild. So if there was a user setting to show all errors, that would be great. 

As far as logging back to admin, much preferable for an actual bug tracker (ex: Sentry).